content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+      "pagerduty": {
+        "service_key": "foobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Since the contact does not have a configuration override for **email** or **slack** the default configuration for those handlers are used instead.
+
+![Multiple Handlers with Single Non-Matching Contact](/images/contact-routing/multiple-handlers/single-non-matching-contact.png)
+
+### Multiple Handlers with Multiple Matching Contacts
+
+In this example we have two handlers configured for our check, **email** and **slack**. We have two contacts being used, **support** and **dev**, and both contacts have configuration override for both **email** and **slack**.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.io"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    },
+    "dev": {
+      "email": {
+        "to": "dev@sensuapp.io"
+      },
+      "slack": {
+        "channel": "#dev"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Since both contacts match and both contacts have configuration overrides for both handlers, four handler events are generated. Two emails will be sent, one for **support** contact _to_ "support@sensuapp.io" and one to **dev** contact _to_ "dev@sensuapp.io". The same is true for the **slack** handler with the **support** contact creating a message in "#support" slack **channel** and the **dev** contact creating a message in "#dev" slack **channel**.
+
+![Multiple Handlers with Multiple Matching Contacts](/images/contact-routing/multiple-handlers/multiple-matching-contacts.png)
+
+### Multiple Handlers with Multiple Non-Matching Contacts
+
+In this example we have two handlers configured for our check, **email** and **slack**. We have two contacts being used, **support** and **dev**, and both contacts have configuration override for **pagerduty** and do not include any override for **slack** or **email**.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email",
+        "slack"
+      ],
+      "contacts": [
+        "support",
+        "dev"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "pagerduty": {
+        "service_key": "supportfoobarkey"
+      }
+    },
+    "dev": {
+      "pagerduty": {
+        "service_key": "devfoobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Since both contacts match and but both contacts do not have configuration overrides for both handlers, two handler events are generated. The default configuration for both handlers will be used
+
+![Multiple Handlers with Multiple Non-Matching Contact](/images/contact-routing/multiple-handlers/multiple-non-matching-contacts.png)
```

Comment:
Same as a few of the images above, has the support email in the picture but should be sent to default.

Reply:
updated image